config/locales/mailer.en.yml

```
@@ -16,26 +16,28 @@ en:
         subject: "Update: Your qualified teacher status (QTS) application"
       application_not_submitted:
         subject:
-          0: Your draft QTS application will be deleted in 2 weeks
-          1: Your draft QTS application will be deleted in 1 week
+          0: Your draft QTS application has not been submitted
+          1: Your draft QTS application is about to be deleted
       application_received:
-        subject: We’ve received your application for qualified teacher status (QTS)
+        subject:
+          true: Your QTS application has been received
+          false: "Your QTS application: Awaiting %{certificate}"
```

```suggestion
          without_document: Your QTS application has been received
          awaiting_document: "Your QTS application: Awaiting %{certificate}"
```

Could we name these something like this, just to make it clearer what the difference between the two are?